CODE_STYLE.md

```diff
@@ -7,6 +7,13 @@
 * [`CONTRIBUTING.md`](CONTRIBUTING.md)
 * [`RELEASES.md`](RELEASES.md)
 
+## Languages
+We prefer Rust.
+
+We have a bunch of Bash and Python scripts that [we want to replace with Rust](https://github.com/rerun-io/rerun/issues/3349).
+
+For configs we like JSON and TOML, and hate YAML.
```

Comment:
`hate` is a strong word and sounds irrational, let's avoid that ;)
```suggestion
For configs we like JSON and TOML, and would like to avoid YAML.
```